### ISSUE TITLE
[Data Issue]: Netherlands: Unknown all mapped to geothermal, when it's actually a mix of various technologies?

### ISSUE THREAD

**Issue by q--** [electricitymaps/electricitymaps-contrib#7386]
### When did this happen?

Observed it today

### What zones are affected?

NL

### What is the problem?

So, current  NL Geothermal production is ~220MW [according to ElectricityMaps](https://github.com/user-attachments/assets/0b4cece5-8965-4127-a4b5-afffcc6dd754), which corresponds with the [production of 'other' at the data source NED.nl](https://github.com/user-attachments/assets/12ee3528-b061-4515-8746-6c10b2e65e69). However, NED.nl makes it clear that this actually a mixture of sources, [mentioning](https://ned.nl/en/node/78) hydropower, biogas and biomass* specifically).

\* Must be dedicated biomass, as biomass co-firing  with coal is already accounted for

**Comment by FelixDQ** [comment]
Thank you for reporting @q--. I took a quick look, and the only thing that we are mapping to geothermal is "Type 9", which they classify as Geothermal here: https://ned.nl/nl/handleiding-api. I'm using the Google translated webpage, so I might be misunderstanding the docs. What is your analysis? Our mapping can be seen here:

https://github.com/electricitymaps/electricitymaps-contrib/blob/61d7447385a3ee3df4f2fa380098d312e4ba4f38/parsers/NED.py#L23-L36

**Comment by q--** [comment]
I don't know about their API (I did notice there's also a `HeatPump` field though, so they're probably supporting or planning to support non-electricity energy).

However, [according to Geothermie Nederland](https://geothermie.nl/wp-content/uploads/GNL-Factsheet-Elektriciteit-uit-aardwarmte.pdf "2022 factsheet"), there is no geothermal electricity production in the Netherlands. Their [locations list](https://geothermie.nl/geothermie/locaties-in-nederland/) also doesn't list any geothermal electricity installations, only heat.

So my suspicion is that this category is actually biogas, dedicated biomass and hydropower (and perhaps other energy sources not mentioned) instead of geothermal. 

(I'm assuming that the category reported as 'geothermal' by ElectricityMaps is the same as the [production of 'other' at the data source NED.nl](https://ned.nl/en/node/78) because they produced the same amount of MW when I checked. Alternatively, it might've been a fluke that they were identical and `9` is actually geothermal heat or something. In any case, I don't think it's geothermal electricity because I don't know of any geothermal electricity plant(s) in the Netherlands.)

**Comment by FelixDQ** [comment]
Thank you, that makes sense. I suggest that we remove it in that case.  I verified with the yearly numbers from Eurostat and Ember, and they indeed don't report any geothermal:

<img width="1144" alt="image" src="https://github.com/user-attachments/assets/738dfd8a-8eff-48fb-8c09-53dd3446e164">

**Comment by q--** [comment]
> I suggest that we remove it in that case.

Only if you can confirm this is indeed not electricity generation, otherwise the type needs to be changed to "unknown" and get a carbon intensity override for hydro + biomass/-gas. I saw the NED website mention that they based those types on the Dutch statistics office CBS, so presumably they would have the share each contributes.

**Comment by FelixDQ** [comment]
Yes, we should look more into ned.nl to ensure that we understand the types. I'm basing my assumption on the fact that we already have higher unknown (and total) production than Eurostat, which would be the most official data source that we usually compare to.

**Comment by annaib00** [comment]
For now I have removed the geothermal/type 9 from the parser, so we do not show Netherlands producing any geothermal electricity.

**Comment by annaib00** [comment]
I am closing this issue now, as it seems that geothermal/type 9 is not used for electricity generation. Because it is under the energy production table on NED.nl site (https://ned.nl/datacatalogus) as well as it is mentioned that there is no geothermal for electricity in the Netherlands (https://www.abnamro.com/research/en/our-research/esg-economist-geothermal-energy-has-great-potential-but-there-are-challenges). If you believe this is still an issue, we can reopen the issue :)
